includes/definitions/discovery/wut.yaml

```diff
@@ -20,13 +20,13 @@ modules:
                     oid: wtWebGraphThermoHygro
                     value: wtWebGraphThermoHygroBinaryTempValue
                     num_oid: '.1.3.6.1.4.1.5040.1.2.42.1.4.1.1.{{ $index }}'
-                    descr: wtWebGraphThermoHygroPortName
+                    descr: 'Temperatur'
```

Comment:
This doesn't seem correct. Isnt the old Oid available? In other case it's misspelled

Reply:
No, this was the reason why I started the PR :-). It is German spelled as on the WebUI (German vendor) and before. But in the meantime I think environment is more clear.